modules/entrezdirect/esummary/main.nf

```diff
@@ -0,0 +1,39 @@
+process ENTREZDIRECT_ESUMMARY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::entrez-direct=16.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/entrez-direct:16.2--he881be0_1':
+        'quay.io/biocontainers/entrez-direct:16.2--he881be0_1' }"
+
+    input:
+    tuple val(meta)
+    val uid
+    val database
```

Comment:
This doesn't make sense here, you would definitely want the `uid` be associated with the meta.

I'll add suggestions to what I mean earlier about `-file`

Reply:
Got it, I will change this as well.